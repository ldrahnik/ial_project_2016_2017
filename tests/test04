B 4 A -2 C 2 D -1 B 3 C
