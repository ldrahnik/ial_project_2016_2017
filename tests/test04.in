B 4 A
A -2 C
C 2 D
D -1 B
B 3 C
