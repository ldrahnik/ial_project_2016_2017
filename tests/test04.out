FloydWarshall: A -(1)> B -(2)> D 
BellmanFord: A -(1)> B -(2)> D
BellmanFord: A -(1)> C -(2)> D
